examples/vue/package.json

```diff
@@ -4,11 +4,11 @@
   "private": true,
   "devDependencies": {
     "react-scripts": "0.9.5",
-    "@storybook/vue": "^3.0.0",
+    "@storybook/vue": "3.0.0-alpha.0",
```

Comment:
@shilman Ok so. Semver will not match a prerelease tag with *. Wee need to specify -tag 

I will use ^3.0.0-alpha.0 to match any of the next versions until we up app/vue version to 3.x ;)